test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/Project.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.integrationtests;
+
+public enum Project {
```

Comment:
This naming may cause some conflicts with org.kie.cloud.openshift.resource.Project. Can we name this enum better? e.g. Kjar

Reply:
Good point, will update